charts/cray-metallb/values.yaml

```diff
@@ -63,3 +63,7 @@ metallb:
     serviceAccount:
       create: true
       name: "speaker"
+    tolerations:
+    - key: "node-role.kubernetes.io/control-plane"
+      operator: "Exists"
+      effect: "NoSchedule"
```

Comment:
In the description I posted the tolerations that were in the pods. I originally had both /master and /control-plane tolerations. However, when I looked at the pod, the /master toleration was written twice. The pods look like this is correct.

Reply:
Ah, you are right.  In the chart it has:
```
      {{- if or .Values.speaker.tolerateMaster .Values.speaker.tolerations }}
      tolerations:
      {{- if .Values.speaker.tolerateMaster }}
      - key: node-role.kubernetes.io/master
        effect: NoSchedule
        operator: Exists
      {{- end }}
      {{- with .Values.speaker.tolerations }}
        {{- toYaml . | nindent 6 }}
      {{- end }}
```
So if `tolerateMaster` is `true` (which is the default), then it appends `Values.speaker.tolerations`.

But looking deeper at the upstream source, we are at version `v0.11.0`, the current version is `v0.14.5` and `v0.13.0` and later already has a fix for this:
```
commit 9834f2ad7827cf1d17c60a27119c4bb3d35744bd
Author: Federico Paolinelli <fpaoline@redhat.com>
Date:   Mon Jun 6 10:28:30 2022 +0200

    Speaker: add toleration for node-role.kubernetes.io/control-plane
    
    We run the speakers also on master nodes, and in 1.24 a new
    "control-plane" taint was added.
    
    Signed-off-by: Federico Paolinelli <fpaoline@redhat.com>
```
We should add `metallb` to the upgrade list for CSM 1.6

Reply:
I'll leave it up to you whether you want to look at upgrading metallb to a version that has the fix, or apply this fix for now and do the metallb upgrade later.  Go ahead and resolve this conversation when you decide.

Reply:
Okay, I just read through the release notes at https://metallb.universe.tf/release-notes, and there are enough changes that we should do the upgrade as a separate issue.  Most notably, a switch from using a ConfigMap to using CRDs to do the configuration.

Reply:
@spillerc-hpe this was discussed in the PET slack channel yesterday. Could we upgrade to the latest metallb in CSM 1.6 if we set up the chart to use 'Pod security admission' instead of PSP? Or is there a reason we need to wait for k8s 1.25?

Reply:
In theory we should be able to transition from PSP to PSA anytime, afaik they can both be active at the same time.  The more we can switch in CSM 1.6, the less we'll have to do in CSM 1.7 prior to doing the k8s 1.25 upgrade.

Reply:
I've been having a separate conversation about PSA vs. kyverno, and I think I've been convinced that we can't just use PSA as a PSP replacement.  So we'd do the metallb upgrade sticking with PSP for now, and then switch over to kyverno PSS.

BUT... for this thread, for now we stick with this change because trying to upgrade metallb that has the fix will be too disruptive and needs to be done as a separate task.